github_connector/tests/test_github_analysis_rule.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (C) 2020: Odoo Community Association (OCA)
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+import logging
+
+from odoo.tests.common import SavepointCase
+
+_logger = logging.getLogger(__name__)
+
+
+class TestGithubAnalysisRule(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestGithubAnalysisRule, cls).setUpClass()
```

Comment:
You can short this in P3:
```suggestion
        super().setUpClass()
```